app/controllers/users_controller.rb

```diff
@@ -1,13 +1,10 @@
 class UsersController < ApplicationController
   def index
-    if current_user
-      @users = User.all
-    else
-      redirect_to user_session_path
-    end
+    @users = User.all
   end
 
   def show
```

Comment:
Please see the error below which occurs when user clicks on a user profile.
![users_show](https://user-images.githubusercontent.com/35504149/148553285-b38fc8ed-265e-4fe9-b979-d7bc49054eed.png)

Reply:
Fixed